### ISSUE TITLE
Error when generate audit - TypeError: this.getAngular2AppName is not a function

### ISSUE THREAD

**Issue by korkutalitasdelen** [hipster-labs/generator-jhipster-entity-audit#94]
```
Welcome to the JHipster Entity Audit Generator! v3.1.0

? Choose which audit framework you would like to use. Custom JHipster auditing (works with SQL)
? Do you want to enable audit for all existing entities? Yes, update all
? Do you want to add an audit log page for entities? Yes
```

i am getting this error

events.js:165
      throw er; // Unhandled 'error' event
      ^

TypeError: this.getAngular2AppName is not a function
    at module.exports.setupGlobalVar (C:\Users\mfrrt\AppData\Roaming\npm\node_modules\generator-jhipster-entity-audit\generators\app\index.js:172:37)
    at Object.<anonymous> (C:\Users\mfrrt\AppData\Roaming\npm\node_modules\generator-jhipster-entity-audit\node_modules\generator-jhipster\node_modules\yeoman-generator\lib\index.js:399:25)
    at C:\Users\mfrrt\AppData\Roaming\npm\node_modules\generator-jhipster-entity-audit\node_modules\run-async\index.js:25:25
    at new Promise (<anonymous>)
    at C:\Users\mfrrt\AppData\Roaming\npm\node_modules\generator-jhipster-entity-audit\node_modules\run-async\index.js:24:19
    at self.env.runLoop.add.completed (C:\Users\mfrrt\AppData\Roaming\npm\node_modules\generator-jhipster-entity-audit\node_modules\generator-jhipster\node_modules\yeoman-generator\lib\index.js:400:11)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5)
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\mfrrt\AppData\Roaming\npm\node_modules\generator-jhipster-entity-audit\node_modules\generator-jhipster\node_modules\yeoman-generator\lib\index.js:406:18)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5)

**Comment by deepu105** [comment]
what is your jHipser version.

**Comment by kwoyna** [comment]
We fixed this by changing this line to "getAngularAppName"  Removed the "2" and it works great!

But how can we see the jhi_entity_audit_event in the generated application?  I do not see it listed in swagger.  And would be nice to be able to page through this and view individual records.  Is that possible?  And thank you for this feature!

**Comment by deepu105** [comment]
@kwoyna can you do a PR with your fix?

I think the features you ask for is not implemented yet, but the endpoint should be available in swagger

**Comment by korkutalitasdelen** [comment]
i tried jhipster 5.0.0 and 4.14.4. i got same error

**Comment by kwoyna** [comment]
Apparently do not have permissions to push / pr

The fix is ./generators/app/index.js

        this.angular2AppName = this.getAngularAppName();

**Comment by pascalgrimaud** [comment]
@kwoyna :
- to contribute to the project, you need to fork this project first
- then clone your fork locally
- do the changes
- then push to YOUR fork, you have the right, as it is your project
- after that, with GitHub, you'll be able to pull request to this project

**Comment by eyupturkyilmaz** [comment]
Thanks @kwoyna 
for answers:
_Apparently do not have permissions to push / pr

The fix is ./generators/app/index.js

    this.angular2AppName = this.getAngularAppName();_

**Comment by ruddell** [comment]
Fixed in #95, thanks @kwoyna
